src/Umbraco.Web.UI.Client/src/packages/core/components/ref-item/ref-item.element.ts

```diff
@@ -1,16 +1,11 @@
-import { html, customElement, css, property, when } from '@umbraco-cms/backoffice/external/lit';
+import { customElement, css, property, type PropertyValues } from '@umbraco-cms/backoffice/external/lit';
 import { UmbElementMixin } from '@umbraco-cms/backoffice/element-api';
-import { UmbTextStyles } from '@umbraco-cms/backoffice/style';
-import { UUIRefElement, UUIRefEvent, UUIRefNodeElement } from '@umbraco-cms/backoffice/external/uui';
+import { UUIRefEvent, UUIRefNodeElement } from '@umbraco-cms/backoffice/external/uui';
 
-@customElement('umb-ref-item')
-export class UmbRefItemElement extends UmbElementMixin(UUIRefElement) {
-	@property({ type: String })
-	name = '';
-
-	@property({ type: String })
-	detail = '';
+const elementName = 'umb-ref-item';
```

Comment:
I think we agreed not to do this any longer? :-D

Reply:
I thought we agreed the opposite, that we would do this. 😂  I'm cool with reverting this, just let me know.

Reply:
yeah, well the idea is to get rid of it in time. so I would say yes, lets not add it in this PR. :-)